src/server/rollbar.js

```diff
@@ -0,0 +1,233 @@
+var util = require('util');
+
+var Client = require('../rollbar');
+var _ = require('../utility');
+var logger = require('./logger');
+
+var transforms = require('./transforms');
+var predicates = require('./predicates');
+
+function Rollbar(accessToken, options, client) {
+  options = options || {};
+  options.accessToken = accessToken;
+  this.context = 'server';
+  this.initialize(options, client);
+}
+
+Rollbar.prototype.global = function(options) {
+  this.client.global(options);
+};
+
+Rollbar.prototype.configure = function(options) {
+  this.options = _.extend(true, {}, this.options, options);
+  this.client.configure(options);
+};
+
+Rollbar.prototype.initialize = function(options, client) {
+  this.options = _.extend(true, {}, this.options, options);
+
+  options.environment = options.environment || process.env.NODE_ENV || 'unspecified';
+
+  this.client = client || new Client(this.context, this.options);
+
+  addTransformsToNotifier(this.client.notifier);
+  addPredicatesToQueue(this.client.queue);
+
+  if (this.options.handleUncaughtExceptions) {
```

Comment:
Looks like this will happen for each instance of the notifier which doesn't seem correct.